lib/alerts/components/AlertsList.js

```diff
@@ -82,24 +97,36 @@ export default class AlertsList extends Component {
                     onClick={visibilityFilterChanged}
                     value={f}
                     key={f}>
-                    {toSentenceCase(f)} <Badge style={{backgroundColor: '#babec0'}}>{filterCounts[f]}</Badge>
+                    {toSentenceCase(f)}{' '}
+                    <Badge
+                      style={{backgroundColor: '#babec0'}}>
+                      {filterCounts[f]}
+                    </Badge>
                   </OptionButton>
                 ))}
               </ButtonGroup>
             </FormGroup>
-            <FormGroup className='form-inline pull-right' controlId='formControlsSelectMultiple'>
+            <FormGroup
+              className='form-inline pull-right'
+              controlId='formControlsSelectMultiple'>
               <ControlLabel>Sort by</ControlLabel>
               {'  '}
               <FormControl
                 componentClass='select'
                 value={sort && `${sort.type}:${sort.direction}`}
                 onChange={this._onSortChange}>
                 {SORT_OPTIONS.map(option => (
```

Comment:
These options could be precalculated outside of this class.

Reply:
Where would you suggest they go?  This seems like a micro-optimization that wouldn't really improve code legibility.

Reply:
Yeah, it's not a major optimization, just seemed a bit wasteful to create a new list of components each time that will be exactly the same.  You could define all of these options as components in the util file.